src/views/tickets/Ticket.js

```diff
@@ -10,6 +10,7 @@ const REFRESH_EVERY_MS = 1000;
 export default class Ticket extends Component {
   constructor(props) {
     super(props);
+    console.log(props);
```

Comment:
Please take out new `console.log`s on the frontend so we don't have to refactor them out later.